src/js/pagination/paging-collection.js

```diff
@@ -46,8 +46,10 @@
             queryParams: {
                 currentPage: 'page',
                 pageSize: 'page_size',
+                totalRecords: 'count',
+                totalPages: 'num_pages',
```

Comment:
@ehteshamkafeel in https://github.com/edx/edx-ui-toolkit/pull/69/ you set `totalRecords ` and `totalPages ` to `null` to prevent them being sent to the server during `GET` requests. So these changes would be incompatible, right?

Why not just set `state` through the documented [`backbone.paginator.PageableCollection.constructor`](https://backbone-paginator.github.io/backbone.paginator/api/#constructor) API?